devops/ansible/roles/rails-application/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This role is home-grown.

Reply:
I want it to be generic. Ideally, we would use the same role in different projects without changing it. The only changes would be in the var config files.

Reply:
Im just thinking of the inside web server, which has both the `inside` rails app and the `real-time` rails app. Both apps need separate capistrano, puma, and nginx configurations. How would we handle that case?

Reply:
I'm not sure at this point. I think the devil will be in the details so we'll need to wait until we get there to address.